data/index.html

```diff
@@ -146,26 +146,34 @@ <h2>SGID Index</h2>
         <p class="section-read-more">
             <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get Connected</a>
         </p>
     </div>
     <div class="grid__col section alternate">
+        <h2>ArcGIS Online Hosted Feature Services</h2>
+        <p class="readable">AGRC also uploads all our curated datasets to <a href="https://utah.maps.arcgis.com">ArcGIS Online</a> as hosted feature services. You can search for these layers in Esri products and add them to your projects. All the SGID hosted feature services' names start with "Utah" (like "Utah Roads") and have a branded thumbnail. They are also shared with a Utah SGID group that matches their category.</p>
+        <p class="readable">Our ArcGIS Online organization also acts as a repository for certain curated SGID datasets that we remove from the Open SGID. Those datasets that we don't anticipate needing to update in the future but are still useful are tagged "static" and shared with the appropriate Utah SGID group. Others that are no longer up-to-date but have value as a snapshot in time (e.g., Census/ACS data or previous tax district boundaries) are tagged as "shelved" and shared with the <a href="http://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9">AGRC Shelf group</a>.</p>
+        <p class="section-read-more">
+            <a href="https://utah.maps.arcgis.com/home/groups.html" class="btn">Browse ArcGIS Online</a>
+        </p>
+    </div>
+    <div class="grid__col section">
         <h2>The SGID Open Data Site</h2>
-        <p class="readable">The <a href="https://opendata.gis.utah.gov">SGID Open Data site</a> leverages Esri's Open Data platform to create a single portal for any GIS resources hosted on Esri's ArcGIS Online or Enterprise Portal solutions. Each item in Open Data contains web service links for consuming the datasets in webmaps and other applications along with direct data download links. We have uploaded all our curated datasets to ArcGIS Online and shared them with the site, and we encourage other agencies and organizations to <a href="{% link gis-data-sharing/open-data/index.md %}">share</a> their datasets with us as well.</p>
+        <p class="readable">The <a href="https://opendata.gis.utah.gov">SGID Open Data site</a> leverages Esri's Open Data platform to create a single portal for GIS resources hosted in ArcGIS Online or ArcGIS Enterprise Portal. Each item in Open Data contains web service links for consuming the datasets in webmaps and other applications along with data download links. We share all our curated datasets through the site and encourage other agencies and organizations to <a href="{% link gis-data-sharing/open-data/index.md %}">share</a> their datasets here as well to help us spread GIS data to the widest possible audience.</p>
```

Comment:
**thought**: this seems to lean more towards how open data works instead of why i should be interested in using it.

Reply:
Yeah, this is part of the "implied when-to-use" I've unintentionally hidden. Probably best to pull that out into the central list of "when to use what."